content/blog/2024-01-24.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "2024-01-24"
+date = 2024-01-24
++++
+
+---
+
+Today is Wednesday, day 024 of 366 (6.55%). We're in week 04 of 52 (7.69%).
+
+## Planning
+
+What I am going to do today:
+
+- [ ] configure [edit snippet page](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client/blob/main/lib/edit_snippet.dart)'s save button
+- [ ] make text fields read only if the user is not the snippet's owner
+
+## Activities executed
+
+Today I was able to get the API running normally. The problem was linked to the static files not loading, an issue that I had been ignoring for a while because I thought static files did not interfere with the API functions. It turns out that when the [`DEBUG` option in the settings.py](https://github.com/OmnicodeSolutions/luisa_drf_tutorial/blob/25db6e9e99a32a6d26f65dddc1d938344dbe8f3f/tutorial/tutorial/settings.py#L26C1-L26C13) file is set to false, Django REST won't handle static files for you anymore, and you'll need a production web server.
```

Comment:
Probably, all we need is to run `collecstatic` if we do not want `DEBUG` to be set.